h2o-docs/src/product/save-and-load-model.rst

```diff
@@ -128,25 +128,31 @@ Importing in R or Python
         original_model <- h2o.glm(x=cols, y = "response", training_frame = data)    
 
         path <- "/path/to/model/directory"
-        mojo_destination <- h2o.download_mojo(model = original_model, path = path)
+        mojo_destination <- h2o.saveMojo(original_model, path = path)
```

Comment:
Just a small question. Why `saveMojo `and not` save_mojo` in R? There is a mix of both styles in R, isn't it? Then, it is ok but looks strange to me. :)

Reply:
one of these is a typo, below we have save_mojo

Reply:
See https://github.com/h2oai/h2o-3/pull/4546#discussion_r413979176